.travis.yml

```
@@ -4,10 +4,10 @@ dist: xenial
 sudo: false
 
 python:
-    - "2.7"
-    - "3.5"
     - "3.6"
     - "3.7"
+    - "3.8"
+    - "3.9"
 
 install:
     - pip install --upgrade pip setuptools
```

It looks like the tests are failing on travis because there is no fortran compiler.
From looking at your pygedm `.travis.yml` these following lines would be needed.

```suggestion  
- sudo apt-get update -qq
- sudo apt-get install gfortran
```